### PR TITLE
fix ES7 rename-search-field.

### PR DESCRIPTION
**Pull request by ereteog**
> Related #https://github.com/advthreat/iroh/issues/5953

The search on source as a text by default does not work on INT after ES7 migration.
After further investigation, the remapping of textual search for searchable token does not work on ES7 because the implementation did not consider the difference between ES5 and ES7 mappings.

<a name="qa">[§](#qa)</a> QA
============================

After ES7 migration on TEST, In the incident manager, search for `Endpoint`, all `Secure Endpoint` incidents should be returned.

### CODE REVIEW

File: src/ctia/features_service.clj [hunk 1]
```diff
@@ -24,6 +24,7 @@
 (tk/defservice features-service
   FeaturesService
   [[:ConfigService get-in-config]]
+  ;;TODO build a proper init to prepare fields only once
```
Comment (ereteog) [inline src/ctia/features_service.clj:27]: @agzam thoughts?

**Review by marioaquino — APPROVED** [review]
LGTM

File: src/ctia/features_service.clj (hunk 1 above)
Comment (agzam) [reply src/ctia/features_service.clj:27]: Just to avoid reading it from the config every time? What's the bottleneck here?

**Review by agzam — APPROVED** [review]

**Comment by jagrutpa** [comment]
This is not available in TEST. This is INT hence marked as QA - Verify later.

**Pull request merged**